src/mumble/CustomElements.cpp

```diff
@@ -137,6 +137,10 @@ QSize ChatbarTextEdit::minimumSizeHint() const {
 QSize ChatbarTextEdit::sizeHint() const {
 	QSize sh = QTextEdit::sizeHint();
 	sh.setHeight(static_cast<qint32>(document()->documentLayout()->documentSize().height()));
+	const QSize minSize = this->minimumSizeHint();
+	if (sh.height() < minSize.height()) {
+		sh.setHeight(minSize.height());
+	}
```

Comment:
```suggestion
	const int minHeight = minimumSizeHint().height();
	const int documentHeight = document()->documentLayout()->documentSize().height();
	sh.setHeight(std::max(minHeight, documentHeight));
```

I think this is more concise and it makes the intent a bit clearer. The code is untested though so it would be great, if you could test it out on your machine to verify that this still fixes the issue :)

Reply:
Nice suggestion, applied and tested.

Thanks for the review!

Reply:
You're welcome :)